projects/data-example-app/src/board/ui/board-ui.component.ts

```diff
@@ -0,0 +1,48 @@
+import { CdkDragDrop } from '@angular/cdk/drag-drop';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import {
+  DeleteStoryDto,
+  Stories,
+  Story,
+  UpdateStoryDto,
+} from '../../state/story';
+
+@Component({
+  selector: 'board-ui',
+  templateUrl: './board-ui.component.html',
+  styleUrls: ['./board-ui.component.scss'],
+})
+export class BoardUiComponent {
+  @Input() stories: Stories[] = [];
+
+  @Output() add = new EventEmitter<Story>();
+
+  @Output() delete = new EventEmitter<DeleteStoryDto>();
+
+  @Output() update = new EventEmitter<UpdateStoryDto>();
+
+  addNew(column: number, stories: Stories): void {
+    // `as any` because of: https://github.com/ngrx/platform/pull/2899
```

Comment:
Is this stil the case?

Reply:
Yes it is to my knowledge, The pr was merged but I don't know if it has been released yet?

Reply:
Yes, these changes are included in the v11 releases 
Side note: the examples are not using the published packages, but are using the sources directly - so we're always on the latest version here 🙂

Reply:
great sounds good! When I wrote this my PR hadn't been merged in yet but I can update this code now!